pkg/controller/cluster/main-controller.go

```diff
@@ -1352,6 +1352,12 @@ func (c *Controller) syncHandler(key string) error {
 			if err != nil {
 				return err
 			}
+		} else {
```

Comment:
@Alevsk In other places [[1](https://github.com/minio/operator/blob/35dabd3030e822dc8049061d4b0d794d1b7b03a0/pkg/resources/jobs/kes-job.go#L48), [2](https://github.com/minio/operator/blob/35dabd3030e822dc8049061d4b0d794d1b7b03a0/pkg/resources/statefulsets/minio-statefulset.go#L324)] we have `else if t.AutoCert()`. Why is this different?